### PR TITLE
Remove generator page

### LINKED ISSUE #2769: Remove Generator page

### PR DESCRIPTION
**Pull request by nonomoho**
close #2769

### CODE REVIEW

File: src/main/webapp/app/router/index.ts [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-    redirect: { name: 'Generator' },
-  },
-  {
-    path: '/generator',
-    name: 'Generator',
-    component: GeneratorVue,
+    redirect: { name: 'Modules' },
   },
   {
     path: '/modules',
```
Comment (DamnClin) [inline src/main/webapp/app/router/index.ts:10]: Remove this route

**Review by DamnClin — COMMENTED** [review]
Need to add header to the module page (and fix the page CSS on height calculation :D

File: src/main/webapp/app/router/index.ts (hunk 1 above)
Comment (nonomoho) [reply src/main/webapp/app/router/index.ts:10]: You want to remove the router?

Comment (DamnClin) [reply src/main/webapp/app/router/index.ts:10]: As you want, not sure it's really needed to redirect to /modules

**Comment by pascalgrimaud** [comment]
yes, this PR should be merged only when there is a navbar in module page including these links: swagger, github, twitter, official website etc

**Comment by nonomoho** [comment]
Here the result:
image

**Comment by nonomoho** [comment]
For information, the header is styled with bootstrap. As we said, we will remove it and rework the header.

File: src/main/webapp/app/module/primary/modules/Modules.vue [hunk 2]
```diff
@@ -75,10 +75,9 @@ $jhipster-lite-box-radius: 6px;
 
 .jhipster-modules-list {
   flex-grow: 1;
+  overflow-y: auto;
```
Comment (DamnClin) [inline src/main/webapp/app/module/primary/modules/Modules.vue:78]: Why changing this behavior? I think keeping the filters when scrolling is a good idea

**Comment by DamnClin** [comment]
Need to rebase to have the form in the right part of the screen :)

File: src/main/webapp/app/module/primary/modules/Modules.vue (hunk 2 above)
Comment (nonomoho) [reply src/main/webapp/app/module/primary/modules/Modules.vue:78]: My bad... Good catch!

File: src/main/webapp/app/module/primary/modules/Modules.vue [hunk 3]
```diff
@@ -78,7 +78,7 @@ $jhipster-lite-box-radius: 6px;
   flex-grow: 1;
 
   &--categories {
-    height: calc(100vh - 70px);
+    height: calc(100vh - 70px - 68.87px);
```
Comment (DamnClin) [inline src/main/webapp/app/module/primary/modules/Modules.vue:81]: Lol, let's avoid pixels fractions

Comment (nonomoho) [reply src/main/webapp/app/module/primary/modules/Modules.vue:81]: Yes it's due to the bootstrap navbar 😞

**Comment by DamnClin** [comment]
Ok, good for me, cc @pascalgrimaud ok for you?

**Comment by pascalgrimaud** [comment]
Looks good, but before merging this, we need to update the guide before: https://github.com/jhipster/jhipster-lite/blob/main/documentation/guides.md
I can do it after work day

**Comment by nonomoho** [comment]
Oh yes, I forget it... Sorry!

**Comment by pascalgrimaud** [comment]
no worry, you couldn't know it ;-)

**Comment by DamnClin** [comment]
Be carefull to rebase this branch before doing the doc so you'll have the already applied module feedback

**Comment by pascalgrimaud** [comment]
let's merge it and update the guide, once the module page is improved
no need to update it now

**Pull request merged**